### DISCUSSION TITLE
butterfly/2.0

### DISCUSSION THREAD

**Discussion by giscus** [LinwoodCloud/website#13] [General]
# butterfly/2.0

https://linwood.dev/butterfly/2.0

**Comment by DjkoopKiller** [comment]
Hello,

I've been using your software and I find it very helpful. I'm running it on Android.

The only issue I've encountered is that there doesn't seem to be a way to set the page format to A4 for printing. Perhaps it's possible, but I haven't been able to find any information about it online.

Could you please let me know if this is possible?

Thank you.

**Reply by CodeDoctorDE** [reply]
Hello,

Nice that you like my app.
To restrict something on the canvas, you can use an area: https://butterfly.linwood.dev/docs/v2/areas/.
To restrict it to the a4 format, just use aspect ratio portrait or landscape on the area tool
